airbyte-webapp/src/core/request/AirbyteRequestService.ts

```diff
@@ -36,6 +36,7 @@ abstract class AirbyteRequestService {
       resultJsonResponse = await response.json();
     } catch (e) {
       // non json result
+      throw new CommonRequestError(response, "non-json response");
```

Comment:
Actually, we do not need to throw error here. This section was ignored intentionally to follow the same flow for all errors.

Reply:
If we don't throw here then we get an NPE in `throw new CommonRequestError(response, resultJsonResponse.message);` because `resultJsonResponse` isn't defined.